### PR TITLE
Book list navbar

### PR DESCRIPTION
**Pull request by shernanm**
## Summary
* Navbar was added 
* The book list was linked with the detail of a book

## Screenshot
image

## Trello Card
* [Trello Card](https://trello.com/c/YHkYmza5)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,37 +1,118 @@
...
-          <h2 class="book-details">Editorial:</h2>
-          <span class="description">Nombre de la editorial</span>
-        </div>
-        <div>
-          <h2 class="book-details">Año de publicación:</h2>
-          <span class="description">Año publicación</span>
-        </div>
-      </div>
+    <nav class="menu">
+      <ul class="menu-list">
```
Comment (CarlosHWolox) [inline index.html]: It's not necesary to use ul just for two items, use somhing simpler

File: index.html [hunk 2]
```diff
@@ -1,37 +1,118 @@
...
+        <li>
+          <a href="#" target="_self">
```
Comment (CarlosHWolox) [inline index.html]: You could add the image as a background image of the a tag you dont use the img tag

**Review by CarlosHWolox — CHANGES_REQUESTED** [review]

File: bookDetails.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```
Comment (CarlosHWolox) [inline bookDetails.html:1]: I dont know why each pr it shows you added this, is this necessary or could be deleted?

File: index.html [hunk 4]
```diff
@@ -1,37 +1,108 @@
...
     <link rel="stylesheet" type="text/css" href="application.css" />
+    <title>HTML & CSS Training</title>
   </head>
-
   <body>
-    <div class="container">
-      <div class="container-img-book">
-        <img class="book-img" src="./assets/book-cover.png" alt="Book cover" />
+    <nav class="menu">
+      <div class="menu-item">
```
Comment (CarlosHWolox) [inline index.html]: this div is unnecesary, remember use the least amountt of elements as possible

**Review by CarlosHWolox — CHANGES_REQUESTED** [review]

File: bookDetails.html (hunk 3 above)
Comment (shernanm) [reply bookDetails.html:1]: This is because I renamed the file, before this was the index

**Review by CarlosHWolox — APPROVED** [review]

**Pull request merged**